Client/Frontend/Strings.swift

```diff
@@ -1071,6 +1071,69 @@ extension String {
+            )
+            public static let TabsNotificationsStatus = MZLocalizedString(
+                "Notifications.TabsNotificationsStatus",
+                tableName: "Settings",
+                value: "This must be turned on to receive tabs.",
+                comment: "This is the description for the setting that toggles Tabs Sent From Other Devices feature in the settings menu under the Notifications section."
+            )
+            public static let SyncSignInNotificationsTitle = MZLocalizedString(
+                "Notifications.SyncSignInNotificationsTitle",
+                tableName: "Settings",
+                value: "Sync Sign In Confirmation",
+                comment: "This is the title for the setting that toggles Sync Sign In Confirmation feature in the settings menu under the Notifications section."
+            )
+            public static let SyncSignInNotificationsStatus = MZLocalizedString(
+                "Notifications.SyncSignInNotificationsStatus",
+                tableName: "Settings",
+                value: "Get notified when your account is signed in on another device.",
+                comment: "This is the description for the setting that toggles Sync Sign In Confirmation feature in the settings menu under the Notifications section."
+            )
+            public static let TipsAndFeaturesNotificationsTitle = MZLocalizedString(
+                "Notifications.TipsAndFeaturesNotificationsTitle",
+                tableName: "Settings",
+                value: "Tips and Features",
+                comment: "This is the title for the setting that toggles Tips and Features feature in the settings menu under the Notifications section."
+            )
+            public static let TipsAndFeaturesNotificationsStatus = MZLocalizedString(
+                "Notifications.TipsAndFeaturesNotificationsStatus",
+                tableName: "Settings",
+                value: "Learn about useful features and how to get the most out of %@.",
+                comment: "This is the description for the setting that toggles Tips and Features feature in the settings menu under the Notifications section."
+            )
+            public static let TurnOnNotificationsTitle = MZLocalizedString(
+                "Notifications.TurnOnNotificationsTitle",
+                tableName: "Settings",
+                value: "Turn on Notifications",
+                comment: "This is the title informating the user needs to turn on notifications in iOS Settings."
+            )
+            public static let TurnOnNotificationsMessage = MZLocalizedString(
+                "Notifications.TurnOnNotificationsTitle",
+                tableName: "Settings",
+                value: "Go to your device Settings to turn on notifications in %@",
+                comment: "This is the title informating the user needs to turn on notifications in iOS Settings."
```

Comment:
Same thing here as well - `informing`. And also the same regarding describing the placeholder.